File: components/generate/src/main/java/org/cloudfoundry/credhub/views/BulkRegenerateResults.java
package org.cloudfoundry.credhub.views;

import java.util.Set;

import com.fasterxml.jackson.annotation.JsonAutoDetect;

@JsonAutoDetect
public class BulkRegenerateResults {
  private Set<String> regeneratedCredentials;

  public Set<String> getRegeneratedCredentials() {
    return regeneratedCredentials;
  }

  public void setRegeneratedCredentials(final Set<String> regeneratedCredentials) {
    this.regeneratedCredentials = regeneratedCredentials;
  }
}

File: components/generate/src/main/kotlin/org/cloudfoundry/credhub/views/BulkRegenerateResults.kt
package org.cloudfoundry.credhub.views

import com.fasterxml.jackson.annotation.JsonAutoDetect

@JsonAutoDetect
class BulkRegenerateResults {
    private lateinit var regeneratedCredentials: Set<String>

    fun getRegeneratedCredentials(): Set<String> = this.regeneratedCredentials

    fun setRegeneratedCredentials(regeneratedCredentials: Set<String>) {
        this.regeneratedCredentials = regeneratedCredentials
    }
}
